tooling/perform-release.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+# The script assumes the remote is named 'origin' but can be configured:
+REMOTE="origin"
+
+# Ask for confirmation before continuing the release
+function confirmOrAbort() {
+    read -p "Do you want to continue? (y/n): " -n 1 -r
+    echo
+    if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+        echo "Aborting."
+        exit 1
+    fi
+}
+
+# Check if current branch is either 'master' or 'release/v*'
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+if [[ "$CURRENT_BRANCH" == "master" ]]; then
+    MINOR_RELEASE=true
+elif [[ "$CURRENT_BRANCH" =~ ^release/v[0-9]+\.[0-9]+$ ]]; then
```

Comment:
```suggestion
elif [[ "$CURRENT_BRANCH" =~ ^release/v[0-9]+\.[0-9]+\.(x|[0-9]+)$ ]]; then
```

This way we support branche names like `release/v1.51.x` or even `release/v1.51.23`

Reply:
From our release guide, the release branch should stay `v1.2.x` and be reused:
<img width="867" height="133" alt="image" src="https://github.com/user-attachments/assets/a8bfcb36-edb5-4208-989c-ae48d2b7bc09" />